content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -349,6 +349,12 @@ When deleting a `User`, its existing posts' `authorUsername` field values will b
 
 When the `username` of a `User` changes, its existing posts' `authorUsername` field values will be set to 'anonymous'.
 
+### Database-specific requirements
+
+MongoDB and SQL Server have specific requirements for referential actions if you have self-relations or cyclic relations in your data model. SQL Server also has specific requirements if you have relations with multiple cascade paths.
```

Comment:
Initially, I was missing links to pages explaining "self-relations" and "cyclic relations" until I followed the link below.

Both terms appear in this current section for the first time and are fully explained in the subpage.

Not sure what's the best approach but perhaps link both terms immediately to each section in the subpage or somehow introduce both here immediately?